tests/network-tests/run-tests.sh

```diff
@@ -7,24 +7,26 @@ cd $SCRIPT_PATH
 rm ./output.json || :
 
 function cleanup() {
-    docker logs joystream-node --tail 15 || :
-    docker stop joystream-node || :
-    docker rm joystream-node || :
-    echo "# Colossus-1 Logs"
-    docker logs colossus-1 --tail 100 || :
-    echo "# Colossus-2 Logs"
-    docker logs colossus-2 --tail 100 || :
-
-    if [ "${NO_STORAGE}" != true ]; then
-      docker-compose -f ../../docker-compose.storage-squid.yml down -v
-    fi
-
-    docker-compose -f ../../docker-compose.yml down -v
-  }
+  docker logs joystream-node --tail 15 || :
+  docker stop joystream-node || :
+  docker rm joystream-node || :
+  echo "# Colossus-1 Logs"
+  docker logs colossus-1 --tail 100 || :
+  echo "# Colossus-2 Logs"
+  docker logs colossus-2 --tail 100 || :
+
+  if [ "${NO_STORAGE}" != true ]; then
+    docker-compose -f ../../docker-compose.storage-squid.yml down -v
+  fi
+
+  docker-compose -f ../../docker-compose.yml down -v
+  docker-compose -f ../../docker-compose.localstack.yml down -v
```

Comment:
```suggestion
    docker-compose -f ../../docker-compose.storage-squid.yml down -v
    docker-compose -f ../../docker-compose.localstack.yml down -v
  fi

  docker-compose -f ../../docker-compose.yml down -v
```

We can only stop localstack if we started it.